src/App.jsx

```diff
@@ -14,12 +14,14 @@ import Footer from "./components/Footer";
 import ViolenceTestPage from "./containers/ViolenceTestPage";
 import JobsTrainingPage from "./containers/JobsTrainingPage";
 import AboutUsPage from "./containers/AboutUsPage";
+import ChatBotSettings from "./components/ChatBotSettings";
 
 const App = () => (
   <Router>
     <div>
       <HelpNavbar />
       <MainNavbar />
+      <ChatBotSettings />
```

Comment:
the work "setting" made me imagine that this is a configuration for the chatbot you going to use it in a chatbot component not a component itself

Reply:
I would appreciate any name suggestion because it cannot be ChatBot and I didn't come up with anything better than ChatBotSettings to describe the component

Reply:
ChatBotMessages maybe ?

Reply:
renamed to ChatBotMessages